app/src/shared/modules/user/services.js

```diff
@@ -5,15 +5,15 @@ import { formatApiUrl } from '$shared/utils/url'
 import type { ApiResult } from '$shared/flowtype/common-types'
 import type { User, PasswordUpdate } from '$shared/flowtype/user-types'
 
-export const getUserData = (): ApiResult<User> => get(formatApiUrl('users', 'me', {
-    noCache: Date.now(),
-}))
-
-export const putUser = (user: User): ApiResult<User> => put(formatApiUrl('users', 'me'), {
-    headers: {
```

Comment:
This was in the wrong place and unnecessary, content type is set to `'application/json'` automatically if `data` is present.